docs/acp/user/intro-to-acp/developing-an-app.md

```diff
@@ -0,0 +1,142 @@
+You need to have a clear understanding of what your app API will provide to others, and always keep that in mind when you design your app.
+
+### Schema-first
+
+Before you start with the development, you should design your API schema files first. Depending on your requirements, you can have an OpenAPI or an AsyncAPI schema file. In this step, you define the SyncAPI endpoints your app will provide to others, the messages you will emit or consume, and the data you expect to work with.
+
+For more information about AsyncAPI schema design, see [Designing your APIs with AsyncAPI](https://www.asyncapi.com/blog/designing_your_apis_with_asyncapi_part_1). 
+
+For more information about OpenAPI schema design, see [Best practices in API design](https://swagger.io/resources/articles/best-practices-in-api-design/).
+
+You can use the following tools to design your APIs:
+- [AsyncAPI Studio](https://studio.asyncapi.com/)
+- [Swagger Editor](https://editor.swagger.io/)
+
+You can also use the wizards provided by Spryker, which will be used by the [SprykerSDK workflow](LINK).
+
+## Build an app
+
+You can build a new app with the help of the SprykerSDK. After you have installed the Spryker SDK, you can run the following commands to start building the app:
+
+{% info_block infoBox "Info" %}
+
+You can skip through the prompts by hitting Enter.
+
+{% endinfo_block %}
+
+```bash
+cd /www/my-app (The local project directory you created before)
+spryker-sdk sdk:init:project --workflow=pbc
+spryker-sdk sdk:workflow:run
+```
+The first line ensures that you are in the correct working directory. The second line initializes the project, and the third line runs the workflow.
+
+Starting from SprykerSDK version 0.3.0, you can use the following set:
+
+```bash
+cd /www/my-app (The local project directory you created before)
+spryker-sdk sdk:workflow:run # and select PBC workflow
+```
+The workflow guides you as much as possible through the process of building an app. Whenever something needs manual interaction, the workflow stops with a message on what you need to do. After you completed the manual step, re-run the workflow with the `spryker-sdk sdk:workflow:run` command. This continues the previously paused workflow.
+
+### Workflow details
+
+The `spryker-sdk sdk:workflow:run` command guides you through the whole process of building an app. This command does the following.
+
+#### 1. Downloads the boilerplate code and adds app definition files
+
+One of the first steps of the workflow is downloading a reduced Spryker project into the working directory. It already contains a lot of code that lets you start implementing your business logic.
+This step can take a few minutes to complete.
+
+{% info_block infoBox "Info" %}
+
+At this point, no dependencies are installed. 
+
+{% endinfo_block %}
+
+There are a couple of files that an app must have. The workflow guides you through creating them.
+
+**Manifest files**
```

Comment:
will it look like a title?
![image](https://user-images.githubusercontent.com/454530/182884070-e62cb4d4-0bc0-4032-8f5a-27bb829addd8.png)

Reply:
Yes, on the linked page you will get more details for the manifest files.

Reply:
no, @stereomon the question was about part `**Manifest files**` it won't be rendered as a title, but just bold text

Reply:
Yes, it should be a title.